app/pb_kits/playbook/_playbook.scss

```diff
@@ -83,3 +83,4 @@
 @import 'pb_time_stacked/time_stacked';
 @import 'pb_weekday_stacked/weekday_stacked';
 @import './utilities/spacing';
+
```

Comment:
This needs styles added. This should add the background colors without any outside dependencies.